src/main/features/response/form/models/statement-of-means/selfEmployed.ts

```diff
@@ -0,0 +1,97 @@
+import { IsDefined, Max, Min, ValidateIf } from 'class-validator'
+import { Serializable } from 'models/serializable'
+import { IsNotBlank } from 'forms/validation/validators/isBlank'
+import { MaxLength } from 'forms/validation/validators/maxLengthValidator'
+import { ValidationConstraints as GlobalValidationConstants } from 'forms/validation/validationConstraints'
+import { Fractions } from 'forms/validation/validators/fractions'
+import * as toBoolean from 'to-boolean'
+import { NumericUtils } from 'common/utils/numericUtils'
+
+export class ValidationErrors {
+  static readonly JOB_TITLE_REQUIRED: string = 'Enter a job title'
+  static readonly JOB_TITLE_TOO_LONG: string = 'You’ve entered too many characters'
+  static readonly ANNUAL_TURNOVER_REQUIRED: string = 'Enter an annual turnover'
+  static readonly ARE_YOU_BEHIND_ON_TAX_REQUIRED: string = 'Select an option'
+  static readonly AMOUNT_YOU_OWE_REQUIRED: string = 'Enter an amount you owe'
+  static readonly REASON_REQUIRED: string = 'Enter a reason'
+  static readonly REASON_TOO_LONG: string = 'You’ve entered too many characters'
+  static readonly AMOUNT_YOU_OWE_NOT_VALID: string = 'Enter a amount, minimum £$constraint1'
+  static readonly INVALID_DECIMALS: string = 'Enter a valid amount, maximum two decimal places'
+  static readonly TOO_MUCH: string = 'Are you sure this is a valid value?'
+}
+
+export class ValidationConstraints {
+  static readonly JOB_TITLE_MAX_LENGTH: number = 100
```

Comment:
I think I've seen this in more than one place, common validation constraints?

Reply:
done